README.md

```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
```

Comment:
It is made to simplify **development and deployment** of data processing pipelines on [GCP](https://cloud.google.com/gcp/)

Reply:
I have removed this sentence. I think that the main features speak for themselves.